tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -232,6 +236,705 @@ def test_bobby_profiler_user_workflow_multi_batch_row_count_range_rule_and_colum
+        auto=True,
+        profiler_config=custom_profiler_config,
+    )
+    assert result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": -21.0,
+        "strict_min": False,
+        "mostly": 1.0,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "profiler_config": custom_profiler_config,
+        "batch_id": "0808e185a52825d22356de2fe00a8f5f",
+    }
+
+    result = validator.expect_column_values_to_be_between(
+        column=column_name,
+        min_value=-21.0,
+        mostly=1.0,
+        result_format="SUMMARY",
+        include_config=True,
+        auto=False,
+    )
+    assert result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": -21.0,
+        "mostly": 1.0,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": False,
+        "batch_id": "0808e185a52825d22356de2fe00a8f5f",
+    }
+
+
+@pytest.mark.skipif(
+    version.parse(np.version.version) < version.parse("1.21.0"),
+    reason="requires numpy version 1.21.0 or newer",
+)
+@freeze_time("09/26/2019 13:42:41")
+def test_bobby_expect_column_values_to_be_between_auto_yes_default_profiler_config_no_custom_profiler_config_yes(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    # If Expectation already has default Rule-Based Profiler configured, delete it for this test.
+    expectation_impl = get_expectation_impl(
+        expectation_name="expect_column_values_to_be_between"
+    )
+    default_profiler_config: Optional[
+        Union[
+            RuleBasedProfilerConfig,
+            Dict[
+                str,
+                Union[
+                    str,
+                    float,
+                    Optional[Dict[str, Any]],
+                    Optional[Dict[str, Dict[str, Any]]],
+                ],
+            ],
+        ]
+    ] = expectation_impl.default_kwarg_values.get("profiler_config")
+    if default_profiler_config:
+        del expectation_impl.default_kwarg_values["profiler_config"]
+
+    assert "profiler_config" not in expectation_impl.default_kwarg_values
+
+    context: DataContext = bobby_columnar_table_multi_batch_deterministic_data_context
+
+    expectation_suite_name: str = f"tmp.profiler_suite_{str(uuid.uuid4())[:8]}"
+    try:
+        # noinspection PyUnusedLocal
+        suite = context.get_expectation_suite(
+            expectation_suite_name=expectation_suite_name
+        )
+    except ge_exceptions.DataContextError:
+        suite = context.create_expectation_suite(
+            expectation_suite_name=expectation_suite_name
+        )
+        print(f'Created ExpectationSuite "{suite.expectation_suite_name}".')
+
+    batch_request: dict
+
+    validator: Validator
+
+    result: ExpectationValidationResult
+
+    custom_profiler_config: dict
+
+    custom_profiler_config = {
```

Comment:
this should be a config type

Reply:
@donaldheppner Again -- this is a custom override, provided by users (who may find plain dictionary more convenient -- at least we have held this pattern for user-facing `batch_request` API).

Reply:
@donaldheppner This got done, based on our conversation.  Thanks!